packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -299,7 +299,7 @@ $alertDurations:
   }
 }
 
-.container.focused .dismiss-progress:after {
+.container.focused .dismiss-progress::after {
```

Comment:
nice catch. 🐟  Should we have an eslint for this?

Reply:
We already have a rule. 🎉 I believe this got fixed after merging `dev`.